### PR TITLE
Migrate to AsyncAPI 2.0.0

### PR DESCRIPTION
**Pull request by michal-hudy**
**Description**

Changes proposed in this pull request:

- Migrated to AsyncAPI 2.0.0

**Related issue(s)**

#5467

### CODE REVIEW

File: components/cms-services/deploy/asyncapi/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,7 @@
-FROM alpine:3.10
+FROM gcr.io/distroless/static:latest
```
Comment (aerfio) [inline components/cms-services/deploy/asyncapi/Dockerfile]: `:latest`?

Comment (michal-hudy) [reply components/cms-services/deploy/asyncapi/Dockerfile]: Yes, there are no semver tags for this image :(. We are also using `latest` in controllers ;)

**Pull request merged**